scripts/build/builders/host.py

```diff
@@ -428,6 +429,10 @@ def __init__(self, root, runner, app: HostApp, board=HostBoard.NATIVE,
             else:
                 self.extra_gn_options.append('chip_enable_dnssd_tests=false')
 
+        if chip_casting_simplified is not None:
+            if chip_casting_simplified:
+                self.extra_gn_options.append('chip_casting_simplified=true')
```

Comment:
```suggestion
        if chip_casting_simplified is not None:
               self.extra_gn_options.append(f'chip_casting_simplified={str(chip_casting_simplified).lower()}')
```

since otherwise you should use true/false. If you want to support none, make sue of all of None/true/false

Reply:
Noted. Made the change in the latest commit.